src/sentry/event_manager.py

```diff
@@ -461,6 +466,12 @@ def save_error_events(
     ) -> Event:
         jobs = [job]
 
+        if is_sample_event(job["event"]):
+            logger.debug(
+                "save_error_events: processing sample event",
+                extra={"event.id": job["event"].id, "project_id": project.id},
```

Comment:
Might be able to add a `"sample_event": True` to all of the loglines you're creating. That way you can filter for all of them when you're analyzing